lib/ansible/module_utils/facts/system/pkg_mgr.py

```diff
@@ -78,10 +79,17 @@ def collect(self, module=None, collected_facts=None):
             if os.path.exists(pkg['path']):
                 pkg_mgr_name = pkg['name']
 
-        if pkg_mgr_name == 'apt' and \
-                os.path.exists('/usr/bin/rpm') and \
-                not os.path.exists('/usr/bin/dpkg'):
-            pkg_mgr_name = 'apt_rpm'
+        # apt is easily installable and supported by distros other than those
+        # that are debian based, this handles some of those scenarios as they
+        # are reported/requested
+        if pkg_mgr_name == 'apt':
+            if collected_facts["ansible_distribution"] == "Fedora":
+                if int(collected_facts["ansible_distribution_major_version"]) < 15:
```

Comment:
In theory, ansible_distribution_major_version can be a string[1] that wont cast to int, so may need a try/except here.

1:  For ex, if all else fails, it can end up as the string 'NA'